lib/spotlight/engine.rb

```diff
@@ -248,17 +248,30 @@ def self.call(template, source = nil)
 
     config.spambot_honeypot_email_field = :email_address
 
-    config.to_prepare do
-      Blacklight::Configuration.try(:initialize_default_configuration) unless Blacklight::Configuration.try(:initialized_default_configuration?)
+    if Blacklight::VERSION > '8'
+      Blacklight::Configuration.default_configuration do
+        # Field containing the last modified date for a Solr document
+        Blacklight::Configuration.default_values[:index].timestamp_field ||= 'timestamp'
 
-      # Field containing the last modified date for a Solr document
-      Blacklight::Configuration.default_values[:index].timestamp_field ||= 'timestamp'
+        # Default configuration for the browse view
+        Blacklight::Configuration.property :browse, default: Blacklight::OpenStructWithHashAccess.new(document_actions: [])
 
-      # Default configuration for the browse view
-      Blacklight::Configuration.default_values[:browse] ||= Blacklight::OpenStructWithHashAccess.new(document_actions: [])
+        Blacklight::Configuration.default_values[:search_state_fields] ||= []
+        Blacklight::Configuration.default_values[:search_state_fields] += %i[id exhibit_id]
+      end
+    else
+      config.to_prepare do
+        Blacklight::Configuration.try(:initialize_default_configuration) unless Blacklight::Configuration.try(:initialized_default_configuration?)
+
+        # Field containing the last modified date for a Solr document
+        Blacklight::Configuration.default_values[:index].timestamp_field ||= 'timestamp'
```

Comment:
this seems the same between the two branches? i keep staring at it but can't see what's different

Reply:
oh it's inside `config.to_prepare`, nvm. i see